app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/debugmenu/DebugMenuScreen.kt

```diff
@@ -0,0 +1,143 @@
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.x8bit.bitwarden.R
+import com.x8bit.bitwarden.data.platform.manager.model.FlagKey
+import com.x8bit.bitwarden.ui.platform.base.util.EventsEffect
+import com.x8bit.bitwarden.ui.platform.base.util.standardHorizontalMargin
+import com.x8bit.bitwarden.ui.platform.components.appbar.BitwardenTopAppBar
+import com.x8bit.bitwarden.ui.platform.components.appbar.NavigationIcon
+import com.x8bit.bitwarden.ui.platform.components.button.BitwardenOutlinedButton
+import com.x8bit.bitwarden.ui.platform.components.scaffold.BitwardenScaffold
+import com.x8bit.bitwarden.ui.platform.components.util.rememberVectorPainter
+import com.x8bit.bitwarden.ui.platform.feature.debugmenu.components.ListItemContent
+import com.x8bit.bitwarden.ui.platform.theme.BitwardenTheme
+
+/**
+ * Top level screen for the debug menu.
+ */
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun DebugMenuScreen(
+    onNavigateBack: () -> Unit,
+    viewModel: DebugMenuViewModel = hiltViewModel(),
+) {
+    val state by viewModel.stateFlow.collectAsStateWithLifecycle()
+
+    EventsEffect(viewModel = viewModel) { event ->
+        when (event) {
+            DebugMenuEvent.NavigateBack -> onNavigateBack()
+        }
+    }
+
+    BitwardenScaffold(
+        topBar = {
+            BitwardenTopAppBar(
+                title = stringResource(R.string.debug_menu),
+                scrollBehavior = TopAppBarDefaults.pinnedScrollBehavior(),
+                navigationIcon = NavigationIcon(
+                    navigationIcon = rememberVectorPainter(R.drawable.ic_back),
+                    navigationIconContentDescription = stringResource(id = R.string.back),
+                    onNavigationIconClick = remember(viewModel) {
+                        {
+                            viewModel.trySendAction(DebugMenuAction.NavigateBack)
+                        }
+                    },
+                ),
+            )
+        },
+    ) { innerPadding ->
+        Column(
+            modifier = Modifier.padding(innerPadding),
+        ) {
+            Spacer(modifier = Modifier.height(16.dp))
+            FeatureFlagContent(
+                featureFlagMap = state.featureFlags,
+                onValueChange = { key, value ->
+                    viewModel.trySendAction(DebugMenuAction.UpdateFeatureFlag(key, value))
+                },
+                onResetValues = {
+                    viewModel.trySendAction(DebugMenuAction.ResetFeatureFlagValues)
+                },
+                modifier = Modifier.standardHorizontalMargin(),
+            )
+        }
+    }
+}
+
+@Composable
+private fun FeatureFlagContent(
+    featureFlagMap: Map<FlagKey<Any>, Any>,
+    onValueChange: (key: FlagKey<Any>, value: Any) -> Unit,
+    onResetValues: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Card(
+        modifier = modifier,
+        elevation = CardDefaults.outlinedCardElevation(),
```

Comment:
I know there are no designs but can we just match the settings screen. We can use the regular `BitwardenWideSwitch`, `BitwardenListHeaderText`, and `BitwardenFilledButton`.

Reply:
:( I meant to put a disclaimer, if you don't like my UI make a ticket to change it after hahah, but I can just roll this with the other changes requested 👍